app/src/main/java/com/sun/unsplash_02/screen/home/adapter/PhotoAdapter.kt

```diff
@@ -63,46 +64,33 @@ class PhotoAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
     }
 
     fun startLoadMore() {
-        listPhotos.add(null)
-        notifyItemInserted(itemCount - 1)
+        if (itemCount > 0) {
+            listPhotos.add(null)
+            notifyItemInserted(itemCount - 1)
+        }
     }
 
     fun stopLoadMore() {
-        listPhotos.removeAt(itemCount - 1)
-        notifyItemRemoved(itemCount)
-    }
-
-    fun setItemClickListener(listener: ItemClickListener) {
-        itemClickListener = listener
+        if (itemCount > 0) {
+            listPhotos.removeAt(itemCount - 1)
+            notifyItemRemoved(itemCount)
+        }
     }
 
-    class PhotoViewHolder(view: View, private val itemClickListener: ItemClickListener?) :
-        RecyclerView.ViewHolder(view), View.OnClickListener {
-
-        private var listener: ItemClickListener? = null
-        private var imageData: Image? = null
+    inner class PhotoViewHolder(view: View) :
```

Comment:
có thể truyền interface vào constructer ViewHolder
```suggestion
    class PhotoViewHolder(view: View) :
```

Reply:
phần này như a Trí review ở trên nên e chuyển qua dùng hof, k dùng interface nữa ạ